### PR TITLE
Feature: README

### PR DESCRIPTION
**Pull request by vadymmarkov**
@zenangst @RamonGilabert @onmyway133 
When it's reviewed and merged `Malibu` is almost ready to be released. Not sure if we need Demo though.

@zenangst Can we use the cover image you've made? 😄

### CODE REVIEW

**Comment by zenangst** [comment]
@vadymmarkov sure we can! 😁

File: README.md [hunk 1]
```diff
@@ -5,15 +5,481 @@
...
+adds a lot of sugar helpers and moves your code up to the next level:
+
+* No more "callback hell".
+* Your requests are described in one place.
+* Response processing could be easily broken down into multiple logical tasks.
+* Data and errors are handled separately.
+* Your networking code is much cleaner, readable and follows `DRY` principle.
+
+Enjoy the ride, equipped with all necessary gears by `Malibu`, and feel yourself
+like a big wave surfer in sharky waters of asynchronous networking.
```
Comment (zenangst) [inline README.md]: Love the introduction but the flow is a bit off.

What about if you flip it around a bit?

> Equip yourself with the necessary gears of Malibu, become a big wave surfer and let the days of shark infested asynchronous networking be a thing of the past. Enjoy the ride!

File: README.md [hunk 2]
```diff
@@ -5,15 +5,481 @@
 [![Carthage Compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![License](https://img.shields.io/cocoapods/l/Malibu.svg?style=flat)](http://cocoadocs.org/docsets/Malibu)
 [![Platform](https://img.shields.io/cocoapods/p/Malibu.svg?style=flat)](http://cocoadocs.org/docsets/Malibu)
+![Swift](https://img.shields.io/badge/%20in-swift%202.2-orange.svg)
 
 ## Description
 
-**Malibu** description.
+Palm trees, ocean reefs and breaking waves. Welcome to the surf club **Malibu**,
```
Comment (zenangst) [inline README.md]: Does `coral reef` sound more intriguing than `ocean reef`? 😎

File: README.md [hunk 3]
```diff
@@ -5,15 +5,481 @@
...
+It's pretty straightforward to create a new `Networking` instance:
+
+```swift
+// Simple networking with `Default` configuration and no base URL
+let simpleNetworking = Networking()
+
+// More advanced networking
+let networking = Networking(
+  // Every request made on this networking will be scoped by the base URL
+  baseURLString: "http://sharkywaters.com/api/",
```
Comment (zenangst) [inline README.md]: Lovely URL 😁

**Comment by vadymmarkov** [comment]
@zenangst Fixed things you mentioned.

File: README.md [hunk 4]
```diff
@@ -5,15 +5,482 @@
 [![Carthage Compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![License](https://img.shields.io/cocoapods/l/Malibu.svg?style=flat)](http://cocoadocs.org/docsets/Malibu)
 [![Platform](https://img.shields.io/cocoapods/p/Malibu.svg?style=flat)](http://cocoadocs.org/docsets/Malibu)
+![Swift](https://img.shields.io/badge/%20in-swift%202.2-orange.svg)
 
 ## Description
 
-**Malibu** description.
+Palm trees, coral reefs and breaking waves. Welcome to the surf club **Malibu**,
```
Comment (onmyway133) [inline README.md]: 🎸

**Comment by vadymmarkov** [comment]
We are waiting for kick ass logo from @zenangst I guess 😎

**Comment by onmyway133** [comment]
So it happens 👍 🎉

**Pull request merged**